Cannot compile the lastest APSIMX version

Dear all, 

I just downloaded the lastest ApsimX and APSIM.Shared files. Then open ApsimX.sln with my VS2015. Then I compiled APSIM.Shared and copy all files in APSIM.Shared/Bin into ApsimX/ Bin, and copy all files in ApsimX\DeploymentSupport\Windows\Bin64 to  ApsimX/ Bin. Then compile it. It returned me following error. Please kindly let me know what I should do. Thank you very much. 

1>------ Build started: Project: Updater, Configuration: Debug Any CPU ------
2>------ Build started: Project: Models, Configuration: Debug Any CPU ------
3>------ Build started: Project: SWIMFrame, Configuration: Debug Any CPU ------
1>  Updater -> D:\APSIMXJ2\ApsimX\Bin\Updater.exe
3>  SWIMFrame -> D:\APSIMXJ2\ApsimX\Bin\SWIMFrame.exe
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Reporting\ReportResourceBalances.cs(133,31,133,35): error CS1031: Type expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Reporting\ReportResourceBalances.cs(133,31,133,35): error CS1026: ) expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Reporting\ReportResourceBalances.cs(133,35,133,36): error CS1002: ; expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Reporting\ReportResourceBalances.cs(133,35,133,36): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Reporting\ReportResourceBalances.cs(136,14,136,14): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(89,26,89,30): error CS1031: Type expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(89,26,89,30): error CS1026: ) expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(89,30,89,31): error CS1002: ; expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(89,30,89,31): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(92,14,92,14): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(155,31,155,35): error CS1031: Type expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(155,31,155,35): error CS1026: ) expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(155,35,155,36): error CS1002: ; expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(155,35,155,36): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Groupings\RuminantFilter.cs(158,14,158,14): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(120,83,120,87): error CS1031: Type expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(120,83,120,87): error CS1026: ) expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(120,87,120,88): error CS1002: ; expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(120,87,120,88): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(144,37,144,41): error CS1031: Type expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(144,37,144,41): error CS1026: ) expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(144,41,144,42): error CS1002: ; expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(144,41,144,42): error CS1513: } expected
2>D:\APSIMXJ2\ApsimX\Models\CLEM\Resources\RuminantType.cs(164,18,164,18): error CS1513: } expected
4>------ Build started: Project: Importer, Configuration: Debug Any CPU ------
5>------ Build started: Project: APSIMRunner, Configuration: Debug Any CPU ------
4>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Bin\Models.exe' could not be found
5>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Bin\Models.exe' could not be found
6>------ Build started: Project: ApsimNG, Configuration: Debug Any CPU ------
6>C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1820,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors.
6>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Models\Bin\Importer.exe' could not be found
6>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Bin\Models.exe' could not be found
7>------ Build started: Project: UnitTests, Configuration: Debug Any CPU ------
7>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Bin\ApsimNG.exe' could not be found
7>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Models\Bin\Importer.exe' could not be found
7>CSC : error CS0006: Metadata file 'D:\APSIMXJ2\ApsimX\Bin\Models.exe' could not be found
========== Build: 2 succeeded, 5 failed, 1 up-to-date, 0 skipped ==========

--- a/Models/CLEM/Groupings/RuminantFilter.cs
+++ b/Models/CLEM/Groupings/RuminantFilter.cs
@@ -86,7 +86,7 @@ namespace Models.CLEM.Groupings
         public override string ToString()
         {
             string str = "";
-            if (Value is null)
+            if (Value == null)
             {
                 str = "FILTER NOT DEFINED";
             }
@@ -152,7 +152,7 @@ namespace Models.CLEM.Groupings
         /// <returns></returns>
         public override string ModelSummary(bool formatForParentControl)
         {
-            if (this.Value is null)
+            if (this.Value == null)
             {
                 return "<div class=\"errorlink\">[FILTER NOT DEFINED]</div>";
             }

--- a/Models/CLEM/Resources/RuminantType.cs
+++ b/Models/CLEM/Resources/RuminantType.cs
@@ -117,7 +117,7 @@ namespace Models.CLEM.Resources
                 AnimalPriceGroup matchCriteria = null;
                 foreach (AnimalPriceGroup item in Apsim.Children(PriceList, typeof(AnimalPriceGroup)).Cast<AnimalPriceGroup>().Where(a => a.PurchaseOrSale == purchaseStyle | a.PurchaseOrSale == PurchaseOrSalePricingStyleType.Both))
                 {
-                    if (animalList.Filter(item).Count() == 1 & matchIndividual is null)
+                    if (animalList.Filter(item).Count() == 1 & matchIndividual == null)
                     {
                         matchIndividual = item;
                     }
@@ -141,7 +141,7 @@ namespace Models.CLEM.Resources
                     }
                 }
 
-                if(matchCriteria is null)
+                if(matchCriteria == null)
                 {
                     // report specific criteria not found in price list
                     string warningString = "No [" + purchaseStyle.ToString() + "] price entry was found for [r=" + ind.Breed + "] meeting the required criteria [" + property + "]"+ (value.ToUpper() != "TRUE" ? " = [" + value + "]." : ".");

